_format/1.1/index.md

```diff
@@ -1781,6 +1822,183 @@ parameter from this specification, it **MUST** return `400 Bad Request`.
 > Note: This is to preserve the ability of JSON API to make additive additions
 to standard query parameters without conflicting with existing implementations.
 
+## <a href="#profiles" id="profiles" class="headerlink"></a> Profiles
+
+JSON API supports the use of "profiles" as a way to indicate additional
+semantics that apply to a JSON:API request/document, without altering the 
+basic semantics described in this specification.
+
+[RFC 6906](https://tools.ietf.org/html/rfc6906) covers the nature of profile
+identification:
+
+> Profiles are identified by URI. However, as is the case with, for
+  example, XML namespace URIs, the URI in this case only serves as an
+  identifier, meaning that the presence of a specific URI has to be
+  sufficient for a client to assert that a resource representation
+  conforms to a profile. Thus, clients SHOULD treat profile URIs as
+  identifiers and not as links.
+
+However, to aid human understanding, profile URIs **SHOULD** return 
+documentation of the profile.
+
+A profile **MAY** assign meaning to particular elements of the document
+structure, such as resource attributes or members of meta objects, and even
+to query parameters whose behavior is left up to each implementation, such 
+as `filter` and all those that contain a non a-z character.
+
+The scope of a profile **MUST** be clearly delineated. The elements reserved by 
+a profile, and the meaning assigned to those elements, **MUST NOT** change over 
+time or else the profile **MUST** be considered a new profile with a new URI. 
+
+Profiles **MAY** be updated over time to add new capabilities. However, any such 
+changes **MUST** be [backwards and forwards compatible](http://www.w3.org/2001/tag/doc/versioning-compatibility-strategies#terminology),
+in order to not break existing users of the profile. This requirement usually
+limits changes to adding optional keys within objects specified in the profile's
+original definition.
+
+For example, let's say that a profile reserves a `timestamps` member in the
+`meta` object of every resource. Originally, this profile defines the value
+of `timestamps` as an object that must contain one member: `created`. The
+profile could evolve to allow an optional member, `updated`, in the `timestamps`
+object. But it could not make that member required, nor could it introduce
+a new sibling to `timestamps`.
+
+Profiles **SHOULD** reserve at least one object-valued member, and **SHOULD**
```

Comment:
Yeah, this was written with profiles that add data to the document in mind, not those that just use query parameters. Although, on second thought, it's probably a good idea even for a profile like fancy filters to reserve a key (probably somewhere in `meta`) so that, if it ever wants to add some metadata to the response (e.g., the size of the unfiltered set, or a deprecation warning or something), it can do so.

Reply:
We can probably clarify intent here by saying something like:

> In order to support the potential to add features over time, profiles **SHOULD** reserve at least one object-valued member in any portion of the document in which they may realistically expand in the future.